letsencrypt/letsencrypt-pod.yaml.in

```diff
@@ -14,11 +14,4 @@ spec:
     image: gcr.io/@project@/letsencrypt
     ports:
     - containerPort: 80
-    volumeMounts:
-      - mountPath: /etc/letsencrypt
```

Comment:
Why could you delete this secret?

Reply:
That's a volume not a secret.  It's not used anymore.  We just use the letsencrypt secret.